metacat-connector-druid/src/main/java/com.netflix.metacat.connector.druid/IMetacatDruidClient.java

```diff
@@ -0,0 +1,53 @@
+/*
+ *  Copyright 2017 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ */
+
+package com.netflix.metacat.connector.druid;
```

Comment:
Something looks really wrong with your package structure. Your file isn't stored in folders it's actually in one big folder `com.netflix.metacat.connector.druid`.

I suspect you did something screwy with the IDE or something but you need to get the package structure fixed